src/Libraries/CoreNodes/Web.cs

```diff
@@ -28,6 +28,9 @@ public static string WebRequestByUrl(string url)
             // Initialize the WebRequest.
             var myRequest = System.Net.WebRequest.Create(uriResult);
 
+            if (myRequest is System.Net.HttpWebRequest httpRequest)
+                httpRequest.UserAgent = "Dynamo";
```

Comment:
Could you add a comment describing why this is needed?

Reply:
Addressed in 190a1e7